src/routes/boot.ts

```diff
@@ -25,6 +26,11 @@ const excludeProperties = <T, K extends keyof T>(
   return obj;
 };
 
+const getFeatures = async (
```

Comment:
Should we keep the same object schema as flagsmith?

Reply:
Hmm, we can do that as well, but we technically are not building a full blown feature management system now.
Do you think it won't be overkill?

Reply:
I'm suggesting it so we can merge it easily with flagsmith flags so the client will not care of who owns the flags actually. It's just the schema so I don't see it as an overkill for now